x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/empty/index.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import styled, { css } from 'styled-components';
+import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+const EmptyPrompt = styled(EuiEmptyPrompt)`
```

Comment:
What was this needed?

Reply:
Because the default max-width is too short for the text inside

Reply:
Yeah, I would not set the max-width but rather let the text wrap instead. Attempting to make the element wide enough to accommodate the english version of the text show is not a good approach because the text can be localized and thus the length of the message might change.

Reply:
Totally right!